libs/clients/driving-license-book/src/lib/drivingLicenseBookClient.service.ts

```diff
@@ -34,10 +34,15 @@ import {
   schoolForSchoolStaffMapper,
   schoolTypeMapper,
 } from '../utils/mappers'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import type { Logger } from '@island.is/logging'
+
+const LOGTAG = '[driving-license-book-client]'
```

Comment:
Already included in the logger. You don't need to add this to your logged strings.